### DISCUSSION TITLE
what is the proper way to apply a patch to config.plist?

### DISCUSSION THREAD

**Discussion by ws1088** [tylernguyen/x1c6-hackintosh#166] [Q&A]
Hi All, should I append the patch `.plist` to `config.plist` or first see if the key exists and try to merge? Thanks!

**Comment by mgrimace** [comment]
Sorry to resurrect this, but as a copy/paste novice, do you or anyone have a follow-up answer? I have the touchscreen model, how do I add the patch to the 'main' config.plist?

**Reply by mgrimace** [reply]
Following up with my solution. I added the kexts that were referred-to in the touchscreen patch by viewing the file here (voodooi2c and voodooi2chid), put them in the oc/kexts folder. Then, I opened the config.plist using Propertree, file/oc snapshot, which added those entries to the config.
